collections/test.js

```diff
@@ -84,7 +84,7 @@ xdescribe('Final Test', () => {
       expect(calculateTotalSalary()).toEqual(163000);
     });
   });
-  describe('organiseEmployeesByDepartment', () => {
+  xdescribe('organiseEmployeesByDepartment', () => {
```

Comment:
Skipping this test are we? 😏

Reply:
nah, fell asleep. gonna tackle it now and time how long it takes.